chat-plugins/trivia.js

```diff
@@ -1005,8 +1005,7 @@ const commands = {
 	submit: 'add',
 	add: function (target, room, user, connection, cmd) {
 		if (room.id !== 'questionworkshop') return this.errorReply('This command can only be used in Question Workshop.');
-		if (cmd === 'add' && !this.can('mute', null, room) || !target) return false;
-		if (!this.canTalk()) return;
+		if ((cmd === 'add' && !this.can('mute', null, room)) || (cmd === 'submit' && !this.can('broadcast', null, room)) || !target) return false;
```

Comment:
This line is looooong. Break it up into a few different lines

Also, don't remove `if (!this.canTalk(()) return;`

Reply:
You actually don't have to ``return false`` here.

``this.can('mute', null, room)`` (will return an error message if they can't do it)[https://github.com/Zarel/Pokemon-Showdown/blob/master/chat.js#L593].